tests/bash_tests/unit_test.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+import system_tests
+import os
+import unittest
+
+bSkip=not (os.path.isfile(system_tests.unit_tests) or os.path.isfile(system_tests.unit_tests + '.exe'))
```

Comment:
Could potentially use the platform agnostic [`shutil.which()`](https://docs.python.org/3.8/library/shutil.html#shutil.which) and not worry about extensions...

Reply:
I didn't think of that, @kmilos.  I don't know python very well.  This is good enough for now.  It's working a lot better now than it was a couple of hours ago.

When the CI goes green, it's time to merge.  I want to make progress with the release itself today.  It's not due until 2 weeks today.  Nobody gets fired for delivering ahead of schedule.

Reply:
No biggie, it's clear and specific as is, bet to get that RC out the door.